### ISSUE TITLE
[Screen Reader-Help-Hardware]Heading levels are not defined properly on hardware pane.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2547]
**User Experience:**
Users with cognitive disabilities may rely on utilities to outline or summarize content based on headings, labels, and other landmarks.  When these headings and labels are not descriptive of their related content, these utilities can work poorly and degrade the user’s ability to understand and navigate the product or service.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Screen reader: Narrator

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section on the page and select it.
4-Navigate to 'Hardware' control and select it.
5-Navigate to the various controls lying in the pane opened.
6-Verify, Heading levels are defined properly or not on hardware pane.

**Actual Result**
1.Heading levels are not defined properly on hardware pane. Heading level 2 is not defined on page but heading levels 1 and 3 is defined for the headings.
2.Heading level is defined for open documentation in new tab link. 

**Expected Result**
1.Heading levels should be defined properly on hardware pane. Heading levels should not be skipped.
2.Heading level should not be defined for open documentation in new tab link. 

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260} 

**Suggested fix:** 
In all the applicable forms we need to check below-  
1-Headings in HTML range from H1 (the most important) to H6 (the least important). It’s best to reserve Heading1 (H1) on a web page for the title of that page.  
2-Headings don’t need to decrease from 1 to 6 on every page. While you don’t need to use all of the headings, they should still be sequential.  
For example, avoid skips from H2 to H4. 

**Please refer attachment for more details:**
[MAS2.4.6_Hardware_ Heading level is not defined properly.zip](https://github.com/Microsoft/pxt/files/1196298/MAS2.4.6_Hardware_Heading.leavel.is.not.defined.properly.zip)

**Comment by microsoftsam** [comment]
@veler what needs discussion here? It seems to me that all that needs to be done here is the headers on the Hardware documentation page need to be adjusted.

**Comment by veler** [comment]
Oops, my bad ! It has nothing to do with "needs discussion" I just need to merge it.

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug
